examples/public-dns-with-cloud-dns/variables.tf

```diff
@@ -79,3 +79,9 @@ variable "disable_code_saving" {
   description = "Boolean indicating if code saving is disabled"
   default     = false
 }
+
+variable "size" {
```

Comment:
https://github.com/wandb/terraform-google-wandb/pull/91/files#r1486697627
I am expecting the variable to be `size`. @wandb-zacharyblasczyk